ert/storage/_storage.py

```diff
@@ -535,7 +535,9 @@ def init(*, workspace_name: str) -> None:
 
     for special_key in _SPECIAL_KEYS:
         if f"{workspace_name}.{special_key}" in experiment_names:
-            raise ValueError("Storage already initialized")
+            raise ValueError(
```

Comment:
Just a small detail here. Shouldn't this be a different error: eg. `NameError`?

Reply:
`NameError` is very specific, maybe `RuntimeError` is a better match?

Reply:
`RuntimeError` is another option and is indeed more general.